browser/fragments/platform-browser.js

```diff
@@ -21,10 +21,12 @@ var Platform = {
 	addEventListener: window.addEventListener,
 	inspect: JSON.stringify,
 	getRandomValues: (function(crypto) {
+		if (crypto === undefined) {
```

Comment:
@SimonWoolf did you mean something like this?

Reply:
Sure, that should work. can you confirm it works as expected in IE10 etc.?

Reply:
@SimonWoolf Yup. 
Tests keep on passing on Firefox and Chrome.
And with this also on IE10.
Attaching some screenshots for reference.

![ie](https://user-images.githubusercontent.com/348355/35428737-0065ec6e-0271-11e8-960f-fffa127b237c.png)

[chrome.pdf](https://github.com/ably/ably-js/files/1666935/chrome.pdf)

[ff.pdf](https://github.com/ably/ably-js/files/1666936/ff.pdf)